apps/system/js/app_window.js

```diff
@@ -2272,6 +2274,29 @@
     this.setVisible(false);
   };
 
+
+  /**
+   *  Track current SSL state of the browser.
+   */
+  AppWindow.prototype._sslState = '';
+
+  /**
+   * Handle mozbrowsersecuritychange events from the browser
+   */
+  AppWindow.prototype._handle_mozbrowsersecuritychange =
+    function aw__handle_mozbrowsersecuritychange(evt) {
+      var state = this._sslState = evt.detail.state;
+      this.publish('securitychange', state);
+    };
+
+  /**
+   * Get the current SSL state for the browser
+   */
+  AppWindow.prototype.getSecurity = function() {
```

Comment:
getSSLState is fine, I just wanted to avoid getSSL which is ambiguous - you might expect to get the certificate itself or something. I'll update the patch